components/match2/wikis/ageofempires/get_match_group_copy_paste_wiki.lua

```diff
@@ -32,11 +32,11 @@ function WikiCopyPaste.getMatchCode(bestof, mode, index, opponents, args)
 	local maps
 	if mode == Opponent.solo then
```

Comment:
on phone it looks like if and elseif branches are the same

Reply:
True, the difference was moved to the other function